packages/crayons-styles/src/index.js

```diff
@@ -0,0 +1,50 @@
+import crayons from '../dist/design-tokens/crayons.json';
```

Comment:
let's not point to `dist` folder directly inside `src`.

Reply:
The JSON file is created during build. Yes we can move it to a folder outside dist and add the folder to .gitignore. But this looks simpler.

Reply:
please move it to a folder outside dist.